File: algorithms/2129.capitalize-the-title/solution.go
func capitalizeTitle(title string) string {
	words := []string{}

	for _, word := range strings.Split(title, " ") {
		if len(word) > 2 {
			tmp := strings.ToLower(word)
			words = append(words, string(tmp[0]-32)+tmp[1:])
		} else {
			words = append(words, strings.ToLower(word))
		}
	}
	return strings.Join(words, " ")
}

File: algorithms/2129.capitalize-the-title/solution.rb
# @param {String} title
# @return {String}
def capitalize_title(title)
  words = []

  title.split.each do |word|
    tmp = word.downcase
    if word.size > 2
      words << tmp[0].upcase + tmp[1..]
    else
      words << tmp
    end
  end

  words.join(' ')
end
